src/Employer/Employer.Web/Utility.cs

```diff
@@ -75,10 +75,6 @@ public static IList<string> GetValidRoutesForVacancy(Vacancy vacancy)
             if (string.IsNullOrWhiteSpace(vacancy.NumberOfPositions?.ToString()))
                 return validRoutes;
 
-            validRoutes.AddRange(new[] { RouteNames.ShortDescription_Post, RouteNames.ShortDescription_Get });
```

Comment:
You'll need to add ShortDesciprtion this to VacancyHasStartedPartTwo and update tests